services/endpoint/endpoint.service.js

```diff
@@ -72,34 +36,8 @@ module.exports = class Endpoint extends BaseJsonService {
     }
   }
 
-  static render({
-    label,
-    message,
-    color,
-    labelColor,
-    namedLogo,
-    logoSvg,
-    logoColor,
-    logoWidth,
-    logoPosition,
-    style,
-    isError,
-    cacheSeconds,
-  }) {
-    return {
-      isError,
-      label,
-      message,
-      color,
-      labelColor,
-      namedLogo,
-      logoSvg,
-      logoColor,
-      logoWidth,
-      logoPosition,
-      style,
-      cacheSeconds,
-    }
+  static render(props) {
```

Comment:
Would it be more clear to maintain the original parameter definition here? I know there's a lot of properties but wondering if it would be helpful to be able to see what all is involved with the rendering from within the class